test/livebook/hubs_test.exs

```diff
@@ -0,0 +1,70 @@
+defmodule Livebook.HubsTest do
+  use ExUnit.Case
+
+  import Livebook.Fixtures
+
+  alias Livebook.Hubs
+
+  setup do
+    on_exit(&Hubs.clean_hubs/0)
+
+    :ok
+  end
+
+  test "fetch_hubs/0 returns a list of persisted hubs" do
+    fly = create_fly("fly-baz")
+    assert Hubs.fetch_hubs() == [fly]
```

Comment:
Another general note, for this kind of tests it's better to have a looser assertion, like `assert fly in Hubs.fetch_hubs()`, so concurrent modifications are not a problem. The tests here run synchronously, so it's not an issue.

Reply:
The idea is to make sure only one exists and it's the one we've created inside the test.

Reply:
Yeah, my point was that usually the tests are concurrent (not here), so another test may be saving a hub and the list will have more items :)

Reply:
This is something I've been concerned, because if the data from "test 1" is being fetched on "test 2", there's something wrong. I know Ecto uses Sandbox, so I tried to "emulate" the same state, deleting everything after each test.

Reply:
I wouldn't say it's something wrong. For example with sessions we start named processes and fetching asks the supervisor, so concurrent tests create global sessions simultaneously,m. This is totally fine and it just requires the assertions to account for that :)